http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/KubernetesClusterComponentSpec.scala

```diff
@@ -57,52 +58,58 @@ class KubernetesClusterComponentSpec extends FlatSpecLike with TestComponent {
   it should "aggregate all sub tables on get, and clean up all tables on delete" in isolatedDbTest {
     val savedCluster1 = makeKubeCluster(1).save()
     val savedNodepool1 = makeNodepool(2, savedCluster1.id).save()
-    val namespaceSet = Set(namespace1, namespace0)
-    dbFutureValue(namespaceQuery.saveAllForCluster(savedCluster1.id, namespaceSet))
+    val namespaceNames = List(namespace0, namespace1)
+    dbFutureValue(namespaceQuery.saveAllForCluster(savedCluster1.id, namespaceNames))
+    val namespaces = dbFutureValue(namespaceQuery.getAllForCluster(savedCluster1.id))
+    namespaces.map(_.name) shouldEqual namespaceNames
 
-    val getCluster = dbFutureValue(kubernetesClusterQuery.getFullClusterById(savedCluster1.id))
+    val getCluster = dbFutureValue(kubernetesClusterQuery.getMinimalClusterById(savedCluster1.id))
     getCluster shouldEqual Some(
       savedCluster1
-        .copy(
-          nodepools = savedCluster1.nodepools + savedNodepool1,
-          namespaces = namespaceSet
-        )
+        .copy(namespaces = namespaces, nodepools = savedCluster1.nodepools ++ List(savedNodepool1))
     )
 
-    //we expect 5 records to be affected by the delete: 2 namespaces, 2 nodepools, 1 cluster
-    dbFutureValue(kubernetesClusterQuery.markAsDeleted(savedCluster1.id, Instant.now())) shouldBe 5
+    //we expect 3 records to be affected by the delete: 2 nodepools, 1 cluster
+    dbFutureValue(kubernetesClusterQuery.markAsDeleted(savedCluster1.id, Instant.now())) shouldBe 3
   }
 
-  it should "list all clusters" in isolatedDbTest {
-    val savedCluster1 = makeKubeCluster(1).save()
-    val savedCluster2 = makeKubeCluster(2).save()
-    val savedCluster3 = makeKubeCluster(3).save()
-
-    val now = Instant.now()
-    dbFutureValue(kubernetesClusterQuery.markAsDeleted(savedCluster3.id, now))
-
-    //list active
-    val listCluster1 = dbFutureValue(kubernetesClusterQuery.listFullClusters(savedCluster1.googleProject))
-    listCluster1.size shouldBe 2
-    listCluster1 should contain(savedCluster1)
-    listCluster1 should contain(savedCluster2)
-
-    //list deleted
-    val listCluster2 =
-      dbFutureValue(kubernetesClusterQuery.listFullClusters(savedCluster1.googleProject, includeDeleted = true))
-    val getCluster3 = dbFutureValue(kubernetesClusterQuery.getFullClusterById(savedCluster3.id)).get
-    listCluster2.size shouldBe 3
-    listCluster2 should contain(savedCluster1)
-    listCluster2 should contain(savedCluster2)
-    listCluster2 should contain(getCluster3)
-  }
+  //TODO: move to app
```

Comment:
app?

Reply:
ah, this was leftover from when I was originally designing the routes to support create/get/list cluster. I kept this as a reference since the list app test would be similar. This is no longer needed :)